tripleblindmarket/covid-safe-paths
# Feature/creole tests

#### Linked issues:

https://github.com/tripleblindmarket/covid-safe-paths/issues/689
https://github.com/tripleblindmarket/covid-safe-paths/issues/677
https://github.com/tripleblindmarket/covid-safe-paths/issues/522

#### Screenshots:
These are generated by CI via the tests running.

#### How to test
Run end to end tests locally or in CI. Expect each test to run against English and Haitian Creole localized app.
